redisinsight/api/src/common/constants/api.ts

```diff
@@ -1,3 +1,4 @@
 export const API_PARAM_DATABASE_ID = 'dbInstance';
 export const API_HEADER_DATABASE_INDEX = 'ri-db-index';
+export const API_HEADER_WINDOW_ID = 'ri-window-id';
```

Comment:
could you add rename header to start from `x-` please?

Reply:
renamed